_includes/v2.1/zone-configs/create-a-replication-zone-for-a-system-range.md

```diff
@@ -0,0 +1,37 @@
+In addition to the databases and tables that are visible via the SQL interface, CockroachDB stores internal data in what are called system ranges. CockroachDB comes with pre-configured replication zones for some of these ranges:
+
+Zone Name | Description
+----------|-----------------|------------
+`.meta` | The "meta" ranges contain the authoritative information about the location of all data in the cluster.<br><br>Because historical queries are never run on meta ranges and it is advantageous to keep these ranges smaller for reliable performance, CockroachDB comes with a **pre-configured** `.meta` replication zone giving these ranges a lower-than-default `gc.ttlseconds`.<br><br>If your cluster is running in multiple datacenters, it's a best practice to configure the meta ranges to have a copy in each datacenter.
+`.liveness` | The "liveness" range contains the authoritative information about which nodes are live at any given time.<br><br>Just as for "meta" ranges, historical queries are never run on the liveness range, so CockroachDB comes with a **pre-configured** `.liveness` replication zone giving this range a lower-than-default `gc.ttlseconds`.<br><br>If this range is unavailable, the entire cluster will be unavailable, so giving it a high replication factor is strongly recommended.
```

Comment:
I think this recommendation should be in a red warning block if at all possible.  I'm not sure if that will work in a table (dang this primitive tooling), but perhaps bold?  Also this is more of an engineering question, but should we rather recommend users to not mess with this setting at all unless they really know what they are doing?